tests/Microsoft.Identity.Test.Unit.net45/PlatformProxyPerformanceTests.cs

```diff
@@ -42,7 +42,7 @@ private void ValidateMethodPerformance(long maxMilliseconds, string name, Action
             }
         }
 
-        private const long AllowedMilliseconds = 10;
+        private const long AllowedMilliseconds = 20;
```

Comment:
does this mean that we take a performance hit? what is causing this?

Reply:
I think 10ms is just too strict. After all this involves reflection.

Anyway, this is smth that randomly fails every now and then, so I though to bump the threshold

Reply:
ok